addon/mixins/flexberry-maplayer-actions-handler.js

```diff
@@ -304,6 +307,9 @@ export default Ember.Mixin.create({
         layerPath: layerPath,
         layer: layer
       });
+
+      let rootLayer = this.get('model.rootLayer');
```

Comment:
It won't work if there are several maps on the page & they are stored not in model, but somewhere else.
So we should walk up the hierarchy from child layer till root.